src/patternfly/components/Select/select-toggle-typeahead.hbs

```diff
@@ -1,13 +1,15 @@
-{{#if select--ItemIsSelected}}
-	{{#> form-control controlType="input"
-		input="true"
-		form-control--modifier="pf-c-select__toggle-typeahead"
-		form-control--attribute=(concat 'type="text" id="' id '-typeahead" aria-label="Type to filter" value="' select-typeahead--Placeholder '"')}}
-	{{/form-control}}
-{{else}}
-	{{#> form-control controlType="input"
-		input="true"
-		form-control--modifier="pf-c-select__toggle-typeahead"
-		form-control--attribute=(concat 'type="text" id="' id '-typeahead" aria-label="Type to filter" placeholder="' select-typeahead--Placeholder '"')}}
-	{{/form-control}}
-{{/if}}
+<form class="pf-c-select__toggle-form">
```

Comment:
@tlabaj in this PR @mattnolting has added a BEM element for the `<form>` inside of the select typeahead toggle to match the react examples. Do you know if that form will always exist? Or would some users exclude it? I remember seeing an issue about nested `<form>` elements in the select component (I think it was with the checkbox select), and how that might be some sort of problem, but I don't remember what came of that.